.github/README.md

```diff
@@ -286,13 +296,68 @@ type SaveDataItem interface {
 ```
 
 ### How the server works in detail
+
 <img src="../assets/server.png" alt="server sketch"/>
 
 The crawler has done the difficult part of indexing each bundle, now the server is able to simply retrieve the requested data item from the database.
 
 1. A user requests a specific data item with a key. E. g. the user does the following request: `/beacon/blob_sidecars?block_height=1337`
 2. Now the server looksup the data item location for that key. Following our example, the server would call the database adapter with the following arguments: `Get(1337, EthBlobIndexHeight)`
-    - `EthBlobIndexHeight = 0` because the block_height is the first index defined in `EthBlobs`
+   - `EthBlobIndexHeight = 0` because the block_height is the first index defined in `EthBlobs`
 3. Now that we have the data item's location, serves the data item directly.
 4. At this point the server has provided the user with all the necessariy information to query for the on-chain merkle root for that specific data item.
 5. Finally, the user constructs the local merkle root hash based on the provided data item from the server and compares it to the on-chain merkle root.
+
+### Response Structure
+
+The Trustless API response contains the actual data item and the inclusion proof that contains all necessary information to verify the data item on-chain. The actual data of the data item might be wrapped in a custom structure, depending on the indexer. For example, the `Tendermint` indexer wraps the data item in a `JSON-RPC 2.0` response.
```

Comment:
Please change to _[...] information to verify the data item has been validated through KYVE_